QHyper/solvers/dqm/dqm.py

```diff
@@ -0,0 +1,23 @@
+import os
+from typing import Any
+
+from dwave.system import LeapHybridDQMSampler
+from QHyper.problems.base import Problem
+from QHyper.solvers.base import Solver
+from QHyper.solvers.converter import Converter
+
+token = os.environ["DWAVE_API_TOKEN"]
```

Comment:
Global variables should be in uppercase
```suggestion
DWAVE_API_TOKEN = os.environ["DWAVE_API_TOKEN"]
```